### ISSUE TITLE
Update Pattern sticker & Pattern template in Figma (to have variables version) - Preferences Dialog

### ISSUE THREAD

**Issue by yunjungyeh** [jpmorganchase/salt-ds#4273]
Update Pattern template in Figma (to have variables version) - Preferences Dialog

**Comment by bhoppers2008** [comment]
No further updates to be made to retired libraries.

@yunjungyeh please review if this is still required for Salt DS Figma.

**Comment by yunjungyeh** [comment]
Yes as we are still linking the Pattern template file from Salt DS library

**Comment by mark-tate** [comment]
Cardamon Goal: complete by EOS

**Comment by mark-tate** [comment]
Clove Goal: 
Closed/Decision by EOS

**Comment by yunjungyeh** [comment]
Scheduled a call to Review Pattern template guidance with @navkaur76 @dplsek

**Comment by yunjungyeh** [comment]
New version of the sticker updated and published

To-dos after the discussion:
- Update the preference Dialog pattern page to link/point the guidance about Dialog size (which is the medium for Preference Dialog)

- Update Dialog Guide about the setting size for Dialog (including medium dialog - which is preference dialog)

- Preference Dialog details can be added to guide library


- Preference Dialog Responsive info to Site pattern page for now
  - and next bigger task adding a sub section part to Responsiveness Layout grid:  https://www.saltdesignsystem.com/salt/foundations/responsiveness/index (so it can cover all sorts of containers: drawer/ )

- Once vertical Navigation is updated by Darrin, the Preference dialog needs to be updated
